deploy/terraform/README.md

```diff
@@ -0,0 +1,106 @@
+# Creating AWS resources for OSMT with Terraform #
+***
+
+This guild will help you deploy the necessary resources in AWS to stand up an OSMT app.
+
+## Prerequisites ##
+
+Before you begin, you must complete the following prerequisites:
+
+- An AWS account with permissions to create the necessary resources.
+- AWS CLI installed
+
+## Setting up AWS in Terraform ##
+
+To use Terraform to manage and deploy resources and infrastructure to AWS, you will need to use the AWS
+provider. You must configure the provider with the proper credentials before you can use it. This provider
```

Comment:
Please remove hard line feeds and let the text flow as a natural paragraph. This is the intent for a markdown file.